### PR TITLE
Change select to fluent control

### PR DESCRIPTION
**Pull request by vishwam**
1. Change the RM Select component to use the fluent Select component.
2. Change the device group and time window dropdowns in the RM subheader to use the fluent Select control instead of a custom dropdown.
3. Update to the latest fluent controls and css
4. Change the RM button component to match fluent css (migrating to the fluent controls caused too many breaks).
5. Change the input and button width+height to match the fluent size.
6. Remove the _normalize.scss import from index.scss - the fluent shell control supplies the same normalize file with some overrides to match the fluent styling, and importing it directly reverts some of them.

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
-@import 'styles/normalize';
 @import 'styles/font-face-imports';
+@import 'styles/variables';
+
+// @font-face urls don't work with CSS modules, so we need to declare them in a
+// non-module file: https://github.com/css-modules/css-modules/issues/95
+@font-face {
+  font-family: $icon-font-family;
+  src: url("https://static.azureiotcentral.com/ux/static/media/segmdl2.1.61.38af5f02.woff") format('woff'),
+      url("https://static.azureiotcentral.com/ux/static/media/segmdl2.1.61.38af5f02.ttf") format('truetype');
```
Comment (Coord) [inline src/index.scss]: we have to update the .ttf

Comment (vishwam) [reply src/index.scss]: updated

File: src/components/shell/timeIntervalDropdown/timeIntervalDropdown.js [hunk 2]
```diff
@@ -16,7 +18,7 @@ const optionValues = [
 
 export class TimeIntervalDropdown extends Component {
 
-  onChange = (propOnChange) => ({ target: { value: { value } = {} } = {} }) => {
+  onChange = (propOnChange) => (value) => {
```
Comment (Coord) [inline src/components/shell/timeIntervalDropdown/timeIntervalDropdown.js:20]: nit: I think value had a default value of {}, don't know if it matters here

**Review by Coord — APPROVED** [review]

File: src/components/shell/timeIntervalDropdown/timeIntervalDropdown.js (hunk 2 above)
Comment (vishwam) [reply src/components/shell/timeIntervalDropdown/timeIntervalDropdown.js:20]: Synced offline, the value actually defaults to undefined so we're fine here.

File: src/components/shell/deviceGroupDropdown/deviceGroupDropdown.js [hunk 3]
```diff
@@ -22,11 +22,19 @@ export class DeviceGroupDropdown extends Component {
...
+          select: {
+            className: cx('device-group-dropdown-select'),
+          },
+          chevron: {
+            className: cx('device-group-dropdown-chevron'),
+          },
+        }}
         options={this.deviceGroupsToOptions(deviceGroups)}
         value={activeDeviceGroupId}
-        clearable={false}
```
Comment (vincenzocaruso) [inline src/components/shell/deviceGroupDropdown/deviceGroupDropdown.js:29]: is this read only prop needed?

File: src/styles/_normalize.scss [hunk 4]
```diff
@@ -1,447 +0,0 @@
-/*! normalize.css v7.0.0 | MIT License | github.com/necolas/normalize.css */
```
Comment (vincenzocaruso) [inline src/styles/_normalize.scss:1]: not needed anymore?

File: src/components/shell/deviceGroupDropdown/deviceGroupDropdown.js (hunk 3 above)
Comment (vishwam) [reply src/components/shell/deviceGroupDropdown/deviceGroupDropdown.js:29]: Fluent select input is already readonly

Comment (vishwam) [reply src/components/shell/deviceGroupDropdown/deviceGroupDropdown.js:29]: fluent select control is already readonly

File: src/styles/_normalize.scss (hunk 4 above)
Comment (vishwam) [reply src/styles/_normalize.scss:1]: yeah, this was interfering with the normalize.scss coming in from the CSS library, so I've removed it here. We should upgrade the normalize.scss in the library to the latest version (v8)

**Review by vincenzocaruso — APPROVED** [review]

**Pull request merged**